### PR TITLE
Term entry Javascript window clearInterval

### PR DESCRIPTION
**Pull request by RaffyRod**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by RaffyRod** [comment]
@KTom101  @yangc95  this one is ready too.

File: content/javascript/concepts/window-clearInterval/window-clearInterval.md
Comment (SSwiniarski) [inline content/javascript/concepts/window-clearInterval/window-clearInterval.md:3]:
```suggestion
Description: 'Cancels a timer previously stablished by setInterval().'
```

File: content/javascript/concepts/window-clearInterval/window-clearInterval.md
Comment (SSwiniarski) [inline content/javascript/concepts/window-clearInterval/window-clearInterval.md:16]:
```suggestion
**`clearInterval()`** is a window method used to shut down the loop-calling process of a `setInterval()` execution when it must be stopped or aborted.
```

File: content/javascript/concepts/window-clearInterval/window-clearInterval.md
Comment (SSwiniarski) [inline content/javascript/concepts/window-clearInterval/window-clearInterval.md:2]:
```suggestion
Title: 'clearInterval()'
```

File: content/javascript/concepts/window-clearInterval/window-clearInterval.md
Comment (SSwiniarski) [inline content/javascript/concepts/window-clearInterval/window-clearInterval.md:20]:
```suggestion
`clearInterval()` receives a parameter called `intervalId` which is returned from the previously created `setInterval()` timer.
```

File: content/javascript/concepts/window-clearInterval/window-clearInterval.md
Comment (SSwiniarski) [inline content/javascript/concepts/window-clearInterval/window-clearInterval.md:22]:
```suggestion
The below is an example of how `clearInterval()` is composed:
```

File: content/javascript/concepts/window-clearInterval/window-clearInterval.md
Comment (SSwiniarski) [inline content/javascript/concepts/window-clearInterval/window-clearInterval.md:29]: We need an introductory sentence describing what the example does. Also, it doesn't actually stop the timer, it just declares a function to do so.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@RaffyRod I did a first review and left a few comments.

**Comment by RaffyRod** [comment]
@SSwiniarski added suggested changes.

File: content/javascript/concepts/window-clearInterval/window-clearInterval.md
Comment (SSwiniarski) [inline content/javascript/concepts/window-clearInterval/window-clearInterval.md:30]: The example still never stops the timer. Edit it so it actually uses the function you created.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@RaffyRod I did another pass and left a few more comments.

**Comment by RaffyRod** [comment]
@SSwiniarski changes added.

File: content/javascript/concepts/window-clearInterval/window-clearInterval.md
Comment (SSwiniarski) [inline content/javascript/concepts/window-clearInterval/window-clearInterval.md:31]: You're still never calling the function in this example, so the example never executes `clearInterval()`

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@RaffyRod Still need to update the example.

**Review by SSwiniarski — APPROVED** [review]
@RaffyRod Looks good for a second review.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@RaffyRod I've added a couple of comments. Tag me when you've made the changes. Thanks

**Comment by RaffyRod** [comment]
@caupolicandiaz  changes added.

**Review by caupolicandiaz — APPROVED** [review]
@RaffyRod @SSwiniarski Looks good to go.

**Comment by SSwiniarski** [comment]
@RaffyRod, @caupolicandiaz We should wait for #2282 to merge, before merging this one.

**Comment by SSwiniarski** [comment]
@RaffyRod Thanks for your patience waiting for this to merge, and congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/javascript/window/clearInterval

image

**Pull request merged**
